core/src/main/java/org/elasticsearch/bootstrap/BootstrapInfo.java

```diff
@@ -120,4 +120,8 @@ public Object remove(Object key) {
         }
         return SYSTEM_PROPERTIES;
     }
+
+    public static void init() {
```

Comment:
Ah, that's definitely not obvious. A comment should be added for that so that someone doesn't just delete the no-op method later.